plugins/catalog-node/api-report-alpha.md

```diff
@@ -20,6 +23,24 @@ export interface CatalogAnalysisExtensionPoint {
 // @alpha (undocumented)
 export const catalogAnalysisExtensionPoint: ExtensionPoint<CatalogAnalysisExtensionPoint>;
 
+// @alpha (undocumented)
+export interface CatalogPermissionExtensionPoint {
+  // (undocumented)
+  addPermissionRules(
+    ...rules: Array<
+      CatalogPermissionRuleInput | Array<CatalogPermissionRuleInput>
+    >
+  ): void;
+}
+
+// @alpha (undocumented)
+export const catalogPermissionExtensionPoint: ExtensionPoint<CatalogPermissionExtensionPoint>;
+
+// @alpha (undocumented)
+export type CatalogPermissionRuleInput<
+  TParams extends PermissionRuleParams = PermissionRuleParams,
+> = PermissionRule<Entity, EntitiesSearchFilter, 'catalog-entity', TParams>;
```

Comment:
Oh these additions made me think ... we actually accidentally have a too-narrow type on this one, shouldn't be `EntitiesSearchFilter` I think. This isn't your fault and I will look into it separately, just mentioning! :)